python/skupper_router/management/skrouter.json

```diff
@@ -974,9 +974,9 @@
                 },
                 "dataConnectionCount": {
                     "description": "For inter-router connectors only.  The number of parallel data connections to carry streaming data.",
-                    "type": "integer",
+                    "type": "string",
```

Comment:
Would it be too bad to leave this type as integer and just remove the default ? Let the user supply a value for this in the config file and if no value is provided, just default it to `(qdr_core_get_worker_thread_count(core) + 1) / 2;` ?
Specify in the description what we are going to default it to ?